src/AWS.Deploy.CLI/ConsoleUtilities.cs

```diff
@@ -306,6 +307,68 @@ public string AskUserForValue(string message, string defaultValue, bool allowEmp
             return userValue;
         }
 
+        public Dictionary<string, string> AskUserForKeyValue(Dictionary<string, string> environmentVariables)

```

Comment:
Is this method only going to be used to prompt for environment variables or any general key-value pair? If it is general key-value pairs as well then we should change the argument name from `environmentVariables` to something generic.

If it is only used for prompt for environment variables, then change the function name to `AskForEnvironmentVariable`

Reply:
Done